koku/masu/processor/tasks.py

```diff
@@ -737,30 +737,29 @@ def update_openshift_on_cloud(  # noqa: C901
             worker_cache.release_single_task(task_name, cache_args)
 
 
-@celery_app.task(name="masu.processor.tasks.update_all_summary_tables", queue=UPDATE_SUMMARY_TABLES_QUEUE)
-def update_all_summary_tables(start_date, end_date=None):
-    """Populate all the summary tables for reporting.
+@celery_app.task(name="masu.processor.tasks.update_summary_tables_by_provider", queue=UPDATE_SUMMARY_TABLES_QUEUE)
+def update_summary_tables_by_provider(start_date, provider_type, end_date=None):
```

Comment:
I need to update this to do provider_type now that Michael has removed the accessor.